catalogd/Makefile

```diff
@@ -121,7 +121,7 @@ VERSION := $(shell git describe --tags --always --dirty)
 endif
 export VERSION
 
-export VERSION_PKG     := $(shell go list -mod=mod ./internal/version)
+export VERSION_PKG     := $(shell go list -mod=mod ./../internal/version)
```

Comment:
@camilamacedo86 Why not `export VERSION_PKG     := $(shell go list -m)/internal/version` ?

Reply:
Both give the same result right?
Also, note that this fix is temporary anyway, we will move the cmd to the root asap.

Reply:
Anyway done.
I think your away is more pretty :-)